src/NzbDrone.Core/Backup/BackupService.cs

```diff
@@ -203,11 +205,12 @@ private void BackupConfigFile()
             _diskTransferService.TransferFile(configFile, tempConfigFile, TransferMode.Copy);
         }
 
-        private void CreateVersionInfo()
+        private void CreateVersionInfo(DateTime dateNow)
         {
-            var builder = new StringBuilder();
+            var tempFile = Path.Combine(_backupTempFolder, "INFO");
+            var contents = $"v{BuildInfo.Version}\n{dateNow:yyyy-MM-dd HH:mm:ss}";
```

Comment:
Use a string builder to build the contents instead of building a string with new lines manually.